interface/client/lib/helpers/helperFunctions.js

```diff
@@ -50,6 +50,21 @@ Helpers.formatUrl = function(url){
 };
 
 /**
+Sanatizes URLs to prevent phishing and XSS attacks
+
+@method sanitizeUrl
+@param {String} url
+**/
+Helpers.sanitizeUrl = function(url){
+    url = String(url);
+
+    url = url.replace(/[\t\n\r\s]+/g, '');
+    url = url.replace(/^((?:javascript)?(?:data)?[:\/\/]{1,3})/i, 'http://');
```

Comment:
As I've said, I dislike using naive regexps for parsing URLs. I don't have a bypass, but here's a simple false positive:

sanitizeUrl("javascript/jquery.js"); -> http://jquery.js

So if it's ever used to filter/sanitize relative URLs, it will fail if for `data` and `javascript` folders.